extensions/eda/plugins/event_source/aws_sqs_queue.py

```diff
@@ -46,6 +52,14 @@
       - Set to 0 to disable.
     type: int
     default: 2
+  max_number_of_messages:
+    description:
+      - Optional The maximum number of messages to return. Amazon SQS
+      - never returns more messages than this value (however, fewer
+      - messages might be returned). Valid values: 1 to 10.
+      - Default: 1.
```

Comment:
No need to add `Default` here as it is defined as a separate param below. Refer to [this](https://console.redhat.com/ansible/automation-hub/repo/published/ansible/eda/content/eda%2Fplugins%2Fevent_source/alertmanager/) to see how it is displayed.